applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -151,6 +186,12 @@ static void advertising_process(struct k_work *work)
 	} else
 #endif /* CONFIG_BT_BONDABLE */
 	{
+		ret = bt_csis_generate_rsi(csis, csis_rsi);
+		if (ret != 0) {
```

Comment:
There is a mix of 'if (ret)' and 'if (ret != 0)' in this PR. According to a comment from gmarull (https://github.com/nrfconnect/sdk-nrf/pull/9197#discussion_r1021252982) we should use 'if (ret < 0)'